backend/inspirehep/pidstore/minters/base.py

```diff
@@ -145,7 +145,17 @@ def update(cls, object_uuid, data):
     @classmethod
     def delete(cls, object_uuid, data):
```

Comment:
is migrator calling `delete`?

Reply:
Yes it is. Twice actually: https://github.com/inspirehep/inspirehep/blob/c02b88d7a64420854f13770e082be14945a79cd6/backend/inspirehep/migrator/tasks.py#L470-L471 here directly, and in record.create_or_update (as update and create are also calling minter.delete)